Makefile

```diff
@@ -0,0 +1,68 @@
+test:
+	@go test ./...
+
+test-verbose:
+	@go test ./... -v
+
+bench:
+	@go test -bench=.
+
+rpc-test:
+	@go test -v ./rpc -env [mainnet|devnet|testnet|mock]
+
+simple-call:
```

Comment:
I think It would be a goo option to have it.
I have made sure that if the `developer` haven't set the environment correctly, then the correct procedure to run the examples will be printed in the terminal
like this::>
![image](https://github.com/user-attachments/assets/1e628900-4924-40c1-9b0c-ce762582afd4)